HelpMyStreetFE/HelpMyStreetFE/Views/Shared/Components/Report/Report.cshtml

```diff
@@ -0,0 +1,87 @@
+@using HelpMyStreet.Utils.Utils;
+@using HelpMyStreet.Utils.Extensions;
+
+@model HelpMyStreetFE.Models.Account.Report.ReportViewModel
+
+<script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+<script>
+
+
+</script>
+
+<div id="chartHolder">
+    <canvas id="myChart"></canvas>
+</div>
+<script>
+    const colours = ['rgb(255, 99, 132)', 'rgb(255, 159, 64)', 'rgb(255, 205, 86)', 'rgb(75, 192, 192)', 'rgb(54, 162, 235)', 'rgb(153, 102, 255)', 'rgb(201, 203, 207)'];    
+    const chartdata = @Html.Raw(Json.Serialize(@Model.Data));
```

Comment:
This section would be better done by making a call to a an ApiController - which essentially allows a C# object to be transposed into a JavaScript object.

For a simple example, see the call to getCommunities in coveragemap.js, which lands in MapsController.cs.